x-pack/plugins/security_solution/scripts/endpoint/action_responder/utils.ts

```diff
@@ -144,6 +147,40 @@ export const sendEndpointActionResponse = async (
     }
   }
 
+  // For `get-file`, upload a file to ES
+  if (action.command === 'get-file' && !endpointResponse.error) {
+    // Add the file's metadata
+    const fileMeta = await esClient.index<UploadedFile>({
+      index: FILE_STORAGE_METADATA_INDEX,
+      id: `${action.id}.${action.hosts[0]}`,
```

Comment:
@pzl - I think this is right for now, correct?
If anything changes, and you remember, let us know and we'll update.

Reply:
correct for now. This is a completely opaque value that _we_ get to determine. So whatever order or value is easiest for us to look up again.